knowledge-base/kb-restrictfirstaxis-to-chartpanewidth.md

```diff
@@ -0,0 +1,47 @@
+---
+title: How To Restrict the First Axis Label to be Within the Chart Pane Width
+description: "How to restrict the first axis label to be within the Chart Pane width of CartesianChart element."
+type: how-to
+page_title: How to restrict the first axis Label to be within the Chart Pane width of RadCartesianChart for WPF.
```

Comment:
Don't use the words verbs in all description fields (title, page_title and description). Use similar words, so we can cover more words provided to the google search optimization engine.
I am using to the word "restrict". You can use also "cut" and "clip".
Also, no need to use "How to" in all description fields. I suggest you to use it only in `description`. In the other fields you can start directly with the topic. E.g. __Restrict the first axis label to be within the chart plot area of RadCartesianChart__

Also you can try different sentence structures in the different description fields. For example:
* Avoid first axis label to be rendered outside of the plot area clip
* First axis label overlaps opposite axis